Native-Frameworks/Views/CustomColors/BackgroundColor.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+struct BackgroundColor: View {
+    var body: some View {
+        // Setup display for custom color
+        ZStack {
+            Rectangle()
+                .frame(width: 300, height: 300)
```

Comment:
You call more times frame with same width and height probalby this extension can help you 

``` 
extension View {

 func frame(side: CGFloat) -> some View {
     return self.frame(width: side, height: side)
  }
}
```

than you can set frame on rectancgle `Rectangle().frame(side: 300)`

Reply:
called the method `squareFrame` to fit the description correctly.